scripts/spaced_repetition.py

```
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+SOURCE_FILE = Path('AGENTS.md')
+RAW_FILE = Path('AGENTS.raw.md')
+
+original_text = SOURCE_FILE.read_text()
+RAW_FILE.write_text(original_text)
+
+# parse headers and items from RAW_FILE
+lines = original_text.splitlines()
+if not lines:
+    raise SystemExit('AGENTS.md is empty')
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)